crates/orderbook/src/database/orders.rs

```diff
@@ -318,6 +318,11 @@ fn calculate_status(order: &FullOrder) -> OrderStatus {
     if order.valid_to < Utc::now().timestamp() {
         return OrderStatus::Expired;
     }
+    if let Some((_, valid_to)) = order.ethflow_data {
+        if valid_to < Utc::now().timestamp() {
```

Comment:
One cool Rust trick you can do here is:

```suggestion
    if matches!(&order.ethflow_data, Some((_, valid_to)) if valid_to < Utc::now().timestamp()) {
```

Reply:
<img width="977" alt="image" src="https://user-images.githubusercontent.com/7348154/201864007-d99998a7-ee24-498c-8505-99f1e3299a26.png">

Reply:
Oops! Missing closing `)`. Updated.

Reply:
Ah, it's part of the matches! statement. Sorry, that I could not figure it out, not yet sufficiently used to rust syntax. Thanks for helping out.